hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/AttributeSearchResource.kt

```diff
@@ -169,4 +172,34 @@ class AttributeSearchResource(private val attributeSearchService: AttributeSearc
     @ParameterObject @PageableDefault pageable: Pageable,
   ) =
     attributeSearchService.search(request, pageable)
+
+  @GetMapping("/attributes")
+  @Tag(name = "Attribute search")
+  @Operation(
+    summary = "*** WIP - DO NOT USE!!! *** Retrieve all attributes supported by the attribute search",
+    description = "Returns all attributes that can be passed into the attribute search including their type.",
+    security = [SecurityRequirement(name = "ROLE_GLOBAL_SEARCH"), SecurityRequirement(name = "ROLE_PRISONER_SEARCH")],
```

Comment:
IIRC this generates invalid swagger - see https://github.com/ministryofjustice/hmpps-prisoner-search/blob/e40724b2fba1a8bc6113b57e02beeb6134a11eee/hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/config/OpenApiConfiguration.kt#L60